packages/server/src/config.ts

```diff
@@ -47,6 +47,8 @@ export interface MedplumServerConfig {
   keepAliveTimeout?: number;
   vmContextBotsEnabled?: boolean;
   shutdownTimeoutMilliseconds?: number;
+  heartbeatMilliseconds?: number;
+  heartbeatDisabled?: boolean;
```

Comment:
I made this `heartbeatDisabled` instead of `heartbeatEnabled` to make it somewhat implicit that `heartbeat` is enabled by default. I think this could be a useful convention for optional boolean config values, but open to disagreement

Reply:
Yeah, tough one.  I understand where you're coming from.  I don't have strong preferences, but I know there is a school of thought to never use "disabled" in boolean names to avoid double negatives.

We do already have precedence for "enabled" with default true, `registerEnabled`, `logAuditEvents`